core/http-auth-spi/src/main/java/software/amazon/awssdk/http/auth/spi/internal/DefaultSignRequest.java

```diff
@@ -64,10 +64,10 @@ public <T> T property(SignerProperty<T> property) {
     @SdkInternalApi
     protected abstract static class BuilderImpl<B extends Builder<B, PayloadT, IdentityT>, PayloadT,
         IdentityT extends Identity> implements Builder<B, PayloadT, IdentityT> {
-        private SdkHttpRequest request;
-        private PayloadT payload;
-        private IdentityT identity;
-        private final Map<SignerProperty<?>, Object> properties = new HashMap<>();
+        protected SdkHttpRequest request;
+        protected PayloadT payload;
+        protected IdentityT identity;
+        protected Map<SignerProperty<?>, Object> properties = new HashMap<>();
```

Comment:
do these _need_ to change?

Reply:
They do in order to set them in the subclass' `BuilderImpl(...)` method

Reply:
The setter methods could be used instead of making them protected, but `protected` sounds fine too. Though, I think the map should remain `final`. See another comment later where we discuss that.

Reply:
changed.